i2c/sensirion_i2c_hal.c

```diff
@@ -61,14 +62,14 @@ int16_t sensirion_i2c_select_bus(uint8_t bus_idx) {
  * communication.
  */
 void sensirion_i2c_init(void) {
-    // IMPLEMENT
+    // TODO:IMPLEMENT
```

Comment:
I guess the prefix would have to be `sensirion_i2c_hal_*` - I would change it if we're breaking the interface anyway

Reply:
No we are not breaking the interface. The renaming of the software is not breaking any interface, otherwise I wouldn't have done it. Clearly the interface is changed but I would argue since it is an implementation of the i2c hal, the functions of sensirion_sw_i2c should never be called outside of the specific hal implementation provided in the folder with it. Since this is part of embedded-common we changed that at the same time. The only time we break it is when someone uses a old sw i2c implementation file and pastes it in there, but then they have to rename the file as well. So I would see this as a replacement of the old sw i2c with a new one and not a change on the old one. In this case we are allowed to name our functions however we want. 
In the hal case I think it's different since it could very well be that there is some driver which uses those functions directly. So code outside of embedded-common would be affected. 
FYI: In the driver-generator embedded-common the functions are renamed. So the plan is to rename them eventually in the master branch as well. 

If you don't agree with my argument we rather should think about if we want to merge this in to master or driver-generator/main then thinking about renaming the functions as well.

Reply:
no I see, let's go for master